common/app/views/fragments/atoms/snippets/snippetImage.scala.html

```diff
@@ -0,0 +1,25 @@
+@import layout.ContentWidths.MainMedia
+@import views.support.{ImgSrc, RenderClasses}
+
+@(
+    picture: model.ImageMedia,
+    widths: layout.WidthsByBreakpoint =  MainMedia.inline
+)
+
+<picture>
+    @* IE 9 needs this workaround as per https://scottjehl.github.io/picturefill/ *@
+        <!--[if IE 9]><video style="display: none;"><![endif]-->
+
+    @widths.breakpoints.map { breakpointWidth =>
+        <source media="(min-width: @breakpointWidth.breakpoint.minWidth.getOrElse("0")px) and (-webkit-min-device-pixel-ratio: 1.25), (min-width: @breakpointWidth.breakpoint.minWidth.getOrElse("0")px) and (min-resolution: 120dpi)"
+        sizes="@breakpointWidth.width"
+        srcset="@ImgSrc.srcsetForBreakpoint(breakpointWidth, widths.breakpoints, maybePath = None, maybeImageMedia = Some(picture), hidpi = true)" />
+        <source media="(min-width: @breakpointWidth.breakpoint.minWidth.getOrElse("0")px)"
+        sizes="@breakpointWidth.width"
+        srcset="@ImgSrc.srcsetForBreakpoint(breakpointWidth, widths.breakpoints, maybePath = None, maybeImageMedia = Some(picture))" />
+    }
+
+        <!--[if IE 9]></video><![endif]-->
+
+    <img itemprop="contentUrl" src="@ImgSrc.getFallbackUrl(picture)" />
+</picture>
```

Comment:
Note for @guardian/dotcom-platform: Consolidate the different image templates. They all look bizarrely quite similar 🔨 
https://github.com/guardian/frontend/blob/d7a5d4b1647848ff35e9566cc8c6eb0b9c433e34/common/app/views/fragments/image.scala.html#L37-L37
https://github.com/guardian/frontend/blob/d7a5d4b1647848ff35e9566cc8c6eb0b9c433e34/common/app/views/fragments/items/elements/facia_cards/image.scala.html